lib/CLI/Parser.hs

```diff
@@ -588,7 +563,7 @@ parseFilterOp =
 
 parseFilter :: MParser Filter
 parseFilter =
-  parseFilterByName <|> parseFilterByDate
+  try parseFilterByName <|> try parseFilterByDate <|> parseFilterByField
```

Comment:
No need for `try` here. All these formats start with different strings.

Reply:
Tests do not pass without try. `filter field with name 'date' and 'name'` in particular. This happens when field names are equal to 'date' or 'name'.

Reply:
Makes sense :thinking: